guide/onboarding/funding-a-wallet.md

```diff
@@ -22,101 +30,281 @@ image: https://bitcoin.design/assets/images/guide/onboarding/funding-a-wallet/fu
 
-Finally, new users can buy bitcoin directly from other bitcoin users such as friends, other users at a meetup, or intermediary services like [localbitcoins](https://localbitcoins.com/).
+For users who want to [manually open a channel]({{'/guide/onboarding/funding-a-wallet/#manual-channel-opening' | relative_url}}) this screen should be skippable.
 
 </div>
 
-{% include /tip/open.html label="Tip: Teach users how to transfer" icon="info" color="blue" %}
+{% include /tip/open.html label="⚡ Pay-to-open" icon="info" color="blue" %}
+
+Offering a pay-to-open Lightning service allows users to fund their wallet directly with a Lightning payment.
+
+{% include /tip/close.html %}
+
+### Network sync
+
+Sync methods like compact block filters (BIP157/158) to sync to the Bitcoin network take some time to finish. If your application uses them, you should communicate to the user that their funds will not appear until the sync is complete. You may even consider not presenting a payment request until this sync is complete. This can help prevent users from getting confused about their funds not showing up.
+
+It's recommended to start any required syncing as soon as the user opens your application to limit wait times.
+
+## 3. Sending the bitcoin
+
+The next step for the user is to send bitcoin to the wallet.
+
+Once the bitcoin has been sent, notify the user once it's received. This should be followed by notifying the user that their wallet is being setup. The setup process being the confirmation of the first transaction and opening of a new payment channel.
 
-Users funding new wallets from an exchange could potentially come into contact with unknown concepts, such as a bitcoin address, QR code, or transaction confirmations. This is an excellent opportunity to explain the exchange to personal wallet flow.
+{% include /tip/open.html label="⚡ Zero confirmation channels" icon="info" color="blue" %}
+
+Offering a zero confirmation channel Lightning service allows users to use their wallet without having to wait for confirmations.
 
 {% include /tip/close.html %}
 
-{% include fact/pros.html %}
-  - An easy and popular way to acquire bitcoin
-{% include fact/close.html %}
+For beginner users, it's recommended to open a payment channel for them automatically once the initial transaction is confirmed. This way, they don't have to manually find a peer and open a channel.
+
+Once a payment channel is open, the user's wallet is ready to use. As the newly opened channel only has outbound liquidity, they will not be able to receive Lightning payments yet.
+
+{% include /tip/open.html label="⚡ Pay-to-open" icon="info" color="blue" %}
+
+Offering a pay-to-open Lightning service allows users to receive Lighting payments as soon as they fund their wallet.
```

Comment:
This is great, but on top of the page we said "no lightning service"
![Screenshot 2021-12-20 at 18 14 29](https://user-images.githubusercontent.com/36959754/146807142-3b600336-3f13-47f9-9c50-c09191dc0519.png)
, so I am confused a bit 😄

Reply:
See my comment above about auto-channel opens. I don't consider that to be a Lightning service it's something that the app can just be automated to do.

Reply:
I am not sure, to me this still sounds like a type of "service" we're doing on behalf of the user.